app/Enums/HomePageTabs.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum HomePageTabs: string
+{
+    case Feed = 'feed';
+    case Following = 'following';
+    case Trending = 'trending';
```

Comment:
no needed

Reply:
you want me use static strings?

Reply:
I propose later on in another PR We can use this Enum to render the UI element too instead of static.
What's your opinion in that ?

Reply:
Done..